contrib/utilities/check_parameter_files_in_examples.sh

```diff
@@ -45,8 +45,9 @@ lethe_fluid_particles=("examples/unresolved-cfd-dem")
 for folder in ${lethe_fluid_particles[@]}; do
   for file in $(find "$folder" -type f -name "*.prm"); do
     echo $file
-    # If generator is in the file name this is a lethe-particles file
-    if [[ "$file" == *"generator"* ]];then
+    # If "generator" or "particle" is in the filename this is a
+    # lethe-particles file
+    if [[ "$file" == *"generator"* ]] || [[ "$file" == *"particle"* ]];then
```

Comment:
I feel "generator" is really not appropriate for this case. The loading and the settling are proper DEM simulations and not just a particle generator, this is why I added the "particle" term. We could agree on a more general term, something like "dem", in the name of the DEM parameter files in unresolved CFD-DEM tho.

Reply:
Could DEM be used for all the examples than? I just feel that in the long term adding more words in the filename convention could become a problem.

Reply:
Yeah I agree. Instead of creating  a double convention, let's just set a new more general convention. Otherwise we will end up with a plethora of word. We could say like "particles_insertion", "particles" or "dem". At least a single word.

Reply:
I will change it for particles